metron-interface/metron-alerts/src/app/pcap/model/pcap.request.ts

```diff
@@ -17,13 +17,13 @@
  */
 
 export class PcapRequest {
-  startTimeMs: number = 0;
-  endTimeMs: number = 150000000000000000;
-  ipSrcAddr: string = '';
-  ipSrcPort: number;
-  ipDstAddr: string = '';
-  ipDstPort: number;
-  protocol: string = '';
-  packetFilter: string = '';
-  includeReverse: boolean = false;
+  startTimeMs = 0;
+  endTimeMs = 150000000000000000;
+  ipSrcAddr = '';
+  ipSrcPort = 0;
```

Comment:
Why are we defaulting these ports to 0?  A similar change was introduced in https://github.com/apache/metron/pull/1142 that caused a regression because queries were filtering ports with a value of 0 by default.